index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sharof Khudayberdiev's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="http://fonts.cdnfonts.com/css/inter" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+    <header class="toolbar-container">
+        <div class="welcome">
+            <p>Welcome</p>
+        </div>
+        <div class="toggle-wrapper">
+            <div class="icon-wrapper">
+                <i class="fa-solid fa-bars fa-lg"></i>
+            </div>
+        </div>
+    </header>
+    <section class="intro">
+        <div class="greeting">
+            <div>Hey there. I’m Sharof</div>
+            <div class="green">I’m a software developer</div>
+        </div>
+        <p class="intro-description">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <section class="social">
+            <a href="">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="linkedin.com/in/udev21"><i class="fa-brands fa-linkedin fa-xl"></i></a>
+            <a href="github.com/udev-21"><i class="fa-brands fa-github fa-xl"></i></a>
+            <a href="t.me/udev21"><i class="fa-brands fa-telegram fa-xl"></i></a>
+            <a href="#"><i class="fa-brands fa-angellist fa-xl"></i></a>
+        </section>
```

Comment:
- [ ] Kindly, For social icons  uses a `ul` tag to group them, and `li` per icon; each icon has a link `a` tag to  let you connect your social networks with the icons, using `href.`
- [ ] Kindly, try to stick to the design as much as possible ( For social icons there is space between icons) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
For social icons your project look likes : 
![3](https://user-images.githubusercontent.com/93598189/185670033-05dcd238-049b-426b-9e10-b4b47cd15285.PNG)

The Figma Design Look likes : 

![f3](https://user-images.githubusercontent.com/93598189/185670108-db8224c7-307b-4955-87b0-4f4fcb960c7d.PNG)